### ISSUE TITLE
Job Opportunity: Work on git for JS full-time

### ISSUE THREAD

**Issue by samuelstroschein** [isomorphic-git/isomorphic-git#1747]
@jcubic Thanks for offering to post the job offer here.

We (https://github.com/inlang/inlang) are using isomorphic git to power an editor that acts as VSCode + Figma for non-technical users. We are running into numerous limitations with git as a backend that we need to solve. 

Edit: We are currently exploring whether we continue using isomorphic git, building our own solution on isomorphic git, or switch to libgit2. The end goal is identical: provide a git implementation that runs in the browser.   

### The job

The work in progress [RFC "git-sdk requirements"](https://github.com/inlang/inlang/pull/455) goes into more detail on the limitations we face. I invite you to participate in the RFC process. 

- extend git to have a real-time collaboration layer 
- make git run fast in the browser (solve many I/O problems)
- a plugin system to store any kind of file like SQLite in "something like git"
- ... the list goes on

### Requirements

- deep knowledge about git internals
- knowledgeable about WebAssembly
- based in Europe, US might work too

You can contact me at samuel.stroschein@inlang.com or write directly in this issue

![image](https://user-images.githubusercontent.com/35429197/226403127-b9d27f68-048b-4043-a3f4-006c663f5e05.png)

**Comment by samuelstroschein** [comment]
feel free to ask questions in this issue

**Comment by samuelstroschein** [comment]
@jcubic I clarified in the issue that we exploring isomorphic git, a solution on isomorphic git or libgit2 -> WASM right now. 

I checked our email conversation and you are right. I mentioned working on isomorphic git without disclosing that we are evaluating multiple paths we can take right now. One of them is isomorphic git, a custom solution based on isomorphic git (since we don't need an implementation up to git spec), or libgit2. 

Feel free to delete this issue.
